docs/api/features/importances.py

```diff
@@ -46,6 +47,18 @@ def coef_(outpath):
     viz.poof(outpath=outpath)
 
 
+def stacked_coef_(outpath):
```

Comment:
Generates the figure for the documentation.

Reply:
thanks for adding the "stacked image to documentation"